doc/Security.xml

```diff
@@ -947,13 +987,290 @@ Added certificate-based client authentication</revremark>
+                </entry>
+              </row>
+              <row>
+                <entry>
+                  <para>OIDC2AuthorizationCode</para>
+                </entry>
+                <entry>
+                  <para>OpenID Connect authorization code flow per [Open ID Connect Core].</para>
+                  <para>The ServerUri is used as metadata URI, where you can retrieve the endpoint
+                    URIs for authorization, token and JWKS.</para>
+                </entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </table>
+        <para>The following client authentication methods are defined.</para>
+        <table xml:id="table_smg_rzq_swb">
+          <title>Client authentication methods</title>
+          <tgroup cols="2">
+            <colspec colname="c1" colwidth="30*"/>
+            <colspec colname="c2" colwidth="70*"/>
+            <thead>
+              <row>
+                <entry>
+                  <para>Method</para>
+                </entry>
+                <entry>Description</entry>
+              </row>
+            </thead>
+            <tbody valign="top">
+              <row>
+                <entry>
+                  <para>client_secret_basic</para>
+                </entry>
+                <entry>
+                  <para>Use HTTP Authorization header to specify client_secret, see [RFC
+                    6749]</para>
+                </entry>
+              </row>
+              <row>
+                <entry>
+                  <para>client_secret_post</para>
+                </entry>
+                <entry>
+                  <para>Use HTTP POST body to specify client_secret, see [RFC 6749]</para>
+                </entry>
+              </row>
+              <row>
+                <entry>
+                  <para>client_secret_jwt</para>
+                </entry>
+                <entry>
+                  <para>Use a HMAC signed JWT using client_secret as shared secret, see [OpenID
+                    Connect Core]</para>
+                </entry>
+              </row>
+              <row>
+                <entry>private_key_jwt</entry>
+                <entry>
+                  <para>Use PKI signed JWT using private key, see [OpenID Connect Core]</para>
+                </entry>
+              </row>
+              <row>
+                <entry>
+                  <para>tls_client_auth</para>
+                </entry>
+                <entry>
+                  <para>Use PKI certificate to authenticate, see [RFC 8705]</para>
+                </entry>
+              </row>
+              <row>
+                <entry>
+                  <para>self_signed_tls_client_auth</para>
+                </entry>
+                <entry>
+                  <para>Use self-signed certificate to authenticate, see [RFC 8705]</para>
+                </entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </table>
```

Comment:
In this table, its pretty unspecific how to interpret the `ClientSecret` for different client authentication methods.